File: 1. Learn the Basics/1. Pattern/C++/21.cpp
/*

*****
*   *
*   *
*   *
*****

*/

#include <bits/stdc++.h>
using namespace std;

void getStarPattern(int n) {
    // Write your code here
    for(int i=0;i<n;i++){
        for(int j=0;j<n;j++){
            if(i==0||j==0||i==n-1||j==n-1){
                cout << "*";
            }else{
                cout << " ";
            }
        }
        cout << endl;
    }
}


int main() {
	// your code goes here
	int n;
	cin >> n;
	getStarPattern(n);
	return 0;
}

File: 1. Learn the Basics/1. Pattern/Python/21.py
# *****
# *   *
# *   *
# *   *
# *****

    
def getStarPattern(n):
    for i in range(0,n):
        for j in range(0,n):
            if i==0 or j==0 or i==n-1 or j==n-1:
                print("*",end="")
            else:
                print(" ",end="")
        print("\r")

n = int(input())
getStarPattern(n)